pkg/azclient/snapshotclient/zz_generated_client.go

```diff
@@ -58,13 +58,13 @@ func New(subscriptionID string, credential azcore.TokenCredential, options *arm.
 const GetOperationName = "SnapshotsClient.Get"
 
 // Get gets the Snapshot
-func (client *Client) Get(ctx context.Context, resourceGroupName string, resourceName string) (result *armcompute.Snapshot, err error) {
+func (client *Client) Get(ctx context.Context, resourceGroupName string, targetSnapshotName string) (result *armcompute.Snapshot, err error) {
```

Comment:
`targetSnapshotName ` --> `snapshotName `

pls remove `target` word in all parameters

Reply:
These files are generated. Please refer to changes in template.go